Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/ConfirmedStatusCard.kt

```diff
@@ -0,0 +1,77 @@
+package de.rki.coronawarnapp.covidcertificate.person.ui.details.items
+
+import android.view.ViewGroup
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.covidcertificate.person.core.PersonCertificates.AdmissionState
+import de.rki.coronawarnapp.covidcertificate.person.core.PersonCertificates.AdmissionState.ThreeGWithPCR
+import de.rki.coronawarnapp.covidcertificate.person.core.PersonCertificates.AdmissionState.ThreeGWithRAT
+import de.rki.coronawarnapp.covidcertificate.person.core.PersonCertificates.AdmissionState.TwoG
+import de.rki.coronawarnapp.covidcertificate.person.core.PersonCertificates.AdmissionState.TwoGPlusPCR
+import de.rki.coronawarnapp.covidcertificate.person.core.PersonCertificates.AdmissionState.TwoGPlusRAT
+import de.rki.coronawarnapp.covidcertificate.person.ui.details.PersonDetailsAdapter
+import de.rki.coronawarnapp.covidcertificate.person.ui.overview.PersonColorShade
+import de.rki.coronawarnapp.databinding.ConfirmedStatusCardBinding
+import de.rki.coronawarnapp.util.ContextExtensions.getDrawableCompat
+import de.rki.coronawarnapp.util.lists.diffutil.HasPayloadDiffer
+import setTextWithUrl
+
+class ConfirmedStatusCard(parent: ViewGroup) :
+    PersonDetailsAdapter.PersonDetailsItemVH<ConfirmedStatusCard.Item, ConfirmedStatusCardBinding>(
+        layoutRes = R.layout.confirmed_status_card,
+        parent = parent
+    ) {
+
+    override val viewBinding: Lazy<ConfirmedStatusCardBinding> = lazy {
+        ConfirmedStatusCardBinding.bind(itemView)
+    }
+
+    override val onBindData: ConfirmedStatusCardBinding.(
+        item: Item,
+        payloads: List<Any>
+    ) -> Unit = { item, payloads ->
+
+        when (item.admissionState) {
+            is TwoG -> {
+                subtitle.text = context.resources.getString(R.string.confirmed_status_2g_badge)
```

Comment:
Just a personal prefference, I'm not asking you to change it, but it would be nicer to have an extension function over `ConfirmedStatusCardBinding` and set the UI there.